packages/token-lists/src/top-100.ts

```diff
@@ -75,41 +76,45 @@ const getTokens = async () => {
       {
         from: monthAgo,
         till: today,
-        blacklist: blacklist,
+        blacklist,
       }
     );
 
     return ethereum.dexTrades;
   } catch (error) {
-    console.error(`Error when fetching Top100 Tokens by volume for the past 30 days, error: ${error.message}`);
+    return error;
   }
 };
 
 /**
  * Main function.
  * Fetch tokems, build list, save list.
  */
-const main = async () => {
-  const tokens = await getTokens();
+const main = async (): Promise<void> => {
+  try {
```

Comment:
Big diff here is just wrapping it all in try-catch (done to get rid of consistent-return error within getTokens())